Mathlib/MeasureTheory/Measure/Prod.lean

```diff
@@ -553,6 +553,86 @@ theorem prod_eq {μ : Measure α} [SigmaFinite μ] {ν : Measure β} [SigmaFinit
     isPiSystem_measurableSet isPiSystem_measurableSet μ.toFiniteSpanningSetsIn
     ν.toFiniteSpanningSetsIn fun s hs t ht => h s t hs ht
 
+/-- Two finite measures on a product that are equal on products of sets are equal. -/
+lemma ext_prod {α β : Type*} {mα : MeasurableSpace α} {mβ : MeasurableSpace β}
+    {μ ν : Measure (α × β)} [IsFiniteMeasure μ] [IsFiniteMeasure ν]
```

Comment:
You can drop [IsFiniteMeasure \nu]: the two measures have the same total mass, so if mu is finite then so is nu.

Same thing below.